Rationale.md

```diff
@@ -1,5 +1,19 @@
 # Design Rationale
 
+This document describes rationales for WebAssembly's design decisions, acting as
+footnotes to the main design text, keeping the main specification easier to
+read, and making it easier to revisit decisions later without having to plow
+through all the issues and pull requests. This rationale document tries to list
+how decisions were made, and where tradeoffs were made for the sake of language
+ergonomics, portability, performance, security, and Getting Things Done.
+
+WebAssembly was designed incrementally, with multiple implementations being
+pursued concurrently. Design decisions are made with what the authors believe to
+be a solid rationale, but as the MVP stablizes, implementations appear, and we
```

Comment:
I'm not sure it's worth pointing out that the authors believe they have a solid rationale (if they didn't believe that...).  I think it would actually flow nicely to just cut out the first part of the sentence (and also "implementations appear, " which now sounds funny) so that you have "As the MVP stabilizes and we get experience from real-world codebases,  ...".